snd-server/services/sqlite_storage.py

```diff
@@ -28,3 +29,21 @@ async def migrate(self) -> None:
                 logger.info(f"Running migration {migration}")
 
                 await self.db.execute(query=migration.read_text())
+
+    async def get_user(self, username) -> UserInDB | None:
+        query = "SELECT * FROM users WHERE username = :username"
+        record = await self.db.fetch_one(query=query, values={"username": username})
+        if not record:
+            return None
+
+        return UserInDB.parse_obj(record._mapping)
+
+    async def store_user(self, user) -> UserInDB | None:
+        user_in_DB = await self.get_user(user.username)
+        if user_in_DB:
+            return None
+
+        query = "INSERT INTO users(username, email, hashed_password) VALUES (:username, :email, :hashed_password)"
+        await self.db.execute(query=query, values=user.__dict__)
+
+        return await self.get_user(user.username)
```

Comment:
Generally you would want to keep those methods in respective service e.g. `users_service`, so that rest of the code doesn't need to care about sqlite and we don't have to move them around when we want to change the storage. But feel free to can keep it here for now

Reply:
Thanks, I think I better keep it here for now and move later :)